packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -39,6 +41,8 @@ const matchSearch = (npmPackages, search) => {
       },
       { threshold: matchSorter.rankings.WORD_STARTS_WITH, key: 'attributes.description' },
     ],
+    sorter: (items) => items,
+    baseSort: (a, b) => (a.index < b.index ? -1 : 1),
```

Comment:
I don't really understand why these are needed. The `sorter` especially doesn't seem to be doing much of anything 🤔 (admittedly I haven't tried removing them to see what happens 🙃 )